WooCommerce/Classes/ViewRelated/Dashboard/Settings/Help/HelpAndSupportViewController.swift

```diff
@@ -296,6 +309,19 @@ private extension HelpAndSupportViewController {
     func rowAtIndexPath(_ indexPath: IndexPath) -> Row {
         return sections[indexPath.section].rows[indexPath.row]
     }
+
+    /// Opens custom help center URL in a web view
+    ///
+    func launchCustomHelpCenterWebPage() {
```

Comment:
nit: how about adding a URL parameter here and pass the non-nil URL from `helpCenterWasPressed` so that we don't have to guard twice (here and `helpCenterWasPressed`)?
```suggestion
    func launchCustomHelpCenterWebPage(url: URL) {
```

Reply:
Done in https://github.com/woocommerce/woocommerce-ios/pull/7553/commits/1676bb4687280291698e996241b8dc8356b6aa45